packages/step-wizard/WizardStep.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const WizardStep = ({
+  tag,
+  link,
+  complete,
+  active,
+  disabled,
+  clickable,
+  href,
+  children,
+  className: classes,
+}) => {
+  const className = classNames(
+    classes,
+    'stepwizard-step',
+    { complete },
+    { active },
+    { disabled },
+    { 'stepwizard-step-clickable': clickable }
+  );
+
+  const Tag = tag || (link ? 'a' : 'div');
```

Comment:
has mentioned in the other comment, just check `href` here and get rid of `link`. If `link` is `true` without `href` then that leads to a11y issues as `<a>` requires `href`. If you need `<a>` without `href`... that's `<button>`

Reply:
165af0033e304d743aa239f9b39a4acd7b2fe533